backends/dpdk/DpdkXfail.cmake

```diff
@@ -96,6 +96,16 @@ p4c_add_xfail_reason("dpdk"
   testdata/p4_16_samples/pna-example-tcp-connection-tracking-err.p4
   )
 
+p4c_add_xfail_reason("dpdk"
+  "add_entry is not called from a default action"
+  testdata/p4_16_samples/pna-example-addhit.p4
+  )
+
+p4c_add_xfail_reason("dpdk"
+  "add_entry action cannot be default action"
+   testdata/p4_16_samples/pna-example-miss.p4
+   )
+
```

Comment:
I thought that the standard method for testing programs that give compile-time errors is not to add them to xfail, but to create an expected stderr output file, check that in, and the test script compares the expected vs. actual output?  For DPDK, the choice has been made for this expected stderr output file to be called <foo>.p4-error (if I recall correctly), which is DIFFERENT than the expected stderr output file for non-DPDK programs, which use instead <foo>.p4-stderr   The reason for these different suffixes is so that the same P4 source program can have different expected error messages printed to stderr for different back ends, and each compared against a different expected stderr output file.